src/version/107/frontend/nl2go_manager.php

```diff
@@ -145,12 +145,14 @@ public function getCustomers()
 		$subscriber = filter_input(INPUT_POST, 'subscriber', FILTER_VALIDATE_BOOLEAN);
 		$group = filter_input(INPUT_POST, 'group');
 
-        // Get subscribed customer only if group is newsletter subscriber group.
-        if ($group === self::NEWSLETTER_SUBSCRIBER_GROUP_ID) {
-            $customers = $this->getNewsletterSubscribers();
-        } else {
-            $customers = $this->getRealCustomers($subscriber);
-        }
+		// Include real customers.
+		$customers = $this->getRealCustomers($subscriber);
+
+		// Get subscribed customer only if group is newsletter subscriber group.
+		if ($group === self::NEWSLETTER_SUBSCRIBER_GROUP_ID) {
```

Comment:
The query in `getRealCustomers` will fail with an exception if the group is set to "only_subscribers". So it fetches only the newsletter-subscribers in `getNewsletterSubscribers`.
This change is just reverting to how it was before i started working on it. Because i accidentally made changes in the old plugin version.